tachyon/math/polynomials/univariate/univariate_sparse_polynomial_unittest.cc

```diff
@@ -336,10 +336,11 @@ TEST_F(UnivariateSparsePolynomialTest, DivScalar) {
     expected_terms.push_back(terms[i] / scalar);
   }
 
-  Poly actual = poly / scalar;
+  std::optional<Poly> actual = poly / scalar;
+  ASSERT_TRUE(actual);
   Poly expected(Coeffs(std::move(expected_terms)));
-  EXPECT_EQ(actual, expected);
-  poly /= scalar;
+  EXPECT_EQ(*actual, expected);
+  ASSERT_TRUE(poly /= scalar);
```

Comment:
I'm not sure if we can always assure div result to have value.

Reply:
scalar is guaranteed to not be zero!